internal/api/fetch_test_plan.go

```diff
@@ -10,8 +10,8 @@ import (
 
 // FetchTestPlan fetchs a test plan from the server.
 // ErrRetryTimeout is returned if the client failed to communicate with the server after exceeding the retry limit.
-func (c Client) FetchTestPlan(ctx context.Context, suiteSlug string, identifier string) (*plan.TestPlan, error) {
-	url := fmt.Sprintf("%s/v2/analytics/organizations/%s/suites/%s/test_plan?identifier=%s", c.ServerBaseUrl, c.OrganizationSlug, suiteSlug, identifier)
+func (c Client) FetchTestPlan(ctx context.Context, suiteSlug string, identifier string, jobRetryCount int) (*plan.TestPlan, error) {
+	url := fmt.Sprintf("%s/v2/analytics/organizations/%s/suites/%s/test_plan?identifier=%s&job_retry_count=%v", c.ServerBaseUrl, c.OrganizationSlug, suiteSlug, identifier, jobRetryCount)
```

Comment:
@nprizal your PR expects this as a string, not 100% sure how `%v` will present this value here? It's an int

Reply:
You can use `%d` to print an `int` in a string template. Although, `%v` will default to `%d` when passed an `int`. But I'd use `%d` to be explicit.